example/frog.rkt

```diff
@@ -0,0 +1,42 @@
+#lang racket/base
+
+(require frog/params
+         frog/enhance-body
+         racket/contract
+         rackjure/threading
+         xml/xexpr)
+
+(provide init
+         enhance-body
+         clean)
+
+;; Called early when Frog launches. Use this to set parameters defined
+;; in frog/params.
+(define/contract (init)
+  (-> any)
+  (current-scheme/host "http://www.example.com")
+  (current-title "My Blog")
+  (current-author "The Unknown Author"))
+
+;; Called once per post and non-post page, on the contents.
+(define/contract (enhance-body xs)
+  (-> (listof xexpr/c) (listof xexpr/c))
+  ;; Here we pass the xexprs through a series of functions.
+  (~> xs
+      (syntax-highlight #:python-executable "python"
```

Comment:
@tger This flexibility was added in PR #125 for someone juggling Pythons 2 and 3.